overlord/devicestate/handlers.go

```diff
@@ -121,6 +140,16 @@ func (m *DeviceManager) doMarkSeeded(t *state.Task, _ *tomb.Tomb) error {
 		}
 	}
 
+	var whatSeeded *seededSystem
+	if err := t.Get("seed-system", &whatSeeded); err != nil && err != state.ErrNoState {
+		return err
+	}
+	if whatSeeded != nil {
+		// TODO:UC20 what about remodels?
+		if err := m.recordSeededSystem(st, whatSeeded); err != nil {
+			return fmt.Errorf("cannot record the seeded system: %v", err)
+		}
+	}
 	st.Set("seed-time", time.Now())
```

Comment:
let's get this time early here and set it also as "seed-time" in the seeded-systems entry

Reply:
This one too.